src/App.js

```diff
@@ -101,11 +169,34 @@ function App() {
         </p>
       </Card>
 
-      <SearchBar
-        handleSearch={handleSearch}
-        handleClear={handleClear}
-        searchText={searchText}
-      />
+      <Card>
+        <SearchBar
+          handleSearch={handleSearch}
+          handleClear={handleClear}
+          searchText={searchText}
+        />
+        <div className="flex gap-x-2 items-center">
+          <span className="mt-2">Available filters: </span>
+          <div className="m-l-auto  flex flex-wrap gap-x-2 mt-3">
+            <Badge
+              value="Without Oxygen"
+              active={withoutOxygen}
+              handleFilter={handleWithoutOxygen}
+            ></Badge>
+            <Badge
+              value="With Oxygen"
+              active={oxygen}
+              handleFilter={handleOxygen}
+            ></Badge>
+            <Badge value="ICU" active={icu} handleFilter={handleIcu}></Badge>
+            <Badge
+              value="ICU With Ventilator"
+              active={icuwithventilator}
+              handleFilter={handleIcuWithVentilator}
+            ></Badge>
```

Comment:
the global `badges` object can be used to iterate here. instead of hardcoding it here. it's likely that this list will grow.